web/client/components/data/featuregrid/formatters/index.js

```diff
@@ -8,5 +8,5 @@
 const React = require('react');
 
 module.exports = {
-    getFormatter: (desc) => desc.localType === 'boolean' ? ({value} = {}) => value !== undefined ? <span>{value.toString()}</span> : undefined : undefined
+    getFormatter: (desc) => desc.localType === 'boolean' ? ({value} = {}) => value !== undefined ? <span>{value && value.toString()}</span> : undefined : undefined
```

Comment:
This value && value.toString() may not work with value === false. 
You could be able to identify this regression doing unit test. Please do it  (I suggest to check isNil instead of value !== undefined)